src/environment.ts

```diff
@@ -73,7 +74,11 @@ export class SecureEnvironment implements MembraneBroker {
         // getting proxy factories ready per environment so we can produce
         // the proper errors without leaking instances into a sandbox
         const secureEnvFactory = secureGlobalThis.eval(`(${serializedSecureEnvSourceText})`);
-        this.getSecureValue = secureEnvFactory(this);
+        const rawHooks: MarshalHooks = {
```

Comment:
it turns out that our existing wrapping for Reflect.* helped here because we can just pass those wrappers directly to the sandbox.